lib/matplotlib/axes/_axes.py

```diff
@@ -2672,6 +2672,9 @@ def pie(self, x, explode=None, labels=None, colors=None,
         The pie chart will probably look best if the figure and axes are
         square, or the Axes aspect is equal.
         """
+        # This method sets the aspect ratio of the axis to "equal".
```

Comment:
This should not be a comment but part of the docstring. Please rework the note in the doctring to reflect the new behavior.

Reply:
heh, yeah, this should be the Note part of the docstring, not a comment.